tests/modules/nf-core/smncopynumbercaller/main.nf

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SMNCOPYNUMBERCALLER } from '../../../../modules/nf-core/smncopynumbercaller/main.nf' addParams( options: [args: ''] )
+
+workflow test_smncopynumbercaller {
+
+    input = Channel.of([
+        [ id:'test'], // meta map
+        file("http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeUwRepliSeq/wgEncodeUwRepliSeqBg02esG2AlnRep1.bam", checkIfExists: true),
+        file("http://hgdownload.cse.ucsc.edu/goldenPath/hg19/encodeDCC/wgEncodeUwRepliSeq/wgEncodeUwRepliSeqBg02esG2AlnRep1.bam.bai", checkIfExists: true)
```

Comment:
Can you add these files to `nf-core/test-datasets` in the branch `modules`? Or use one of the available BAM files already in there? This way we have control over our own test files instead of having to rely on test datasets from elsewhere